### PR TITLE
Sync with Pro (TEST)

### PR DESCRIPTION
**Pull request by IagoAbal**
OSS repo was last synced with Pro commit 78299ac3a15dc73013158051dffed0378a8ee11b.
    There are 15 commit(s) to sync.
    Syncing...
    * Skipping (empty) 90875c29985 typo
    * Skipping (empty) b8e365fcf29 Merge pull request #1498 from semgrep/the_big_merge_oss_pro
    * 64c9364fbee feat: AWS validator type for Secrets (#1483)
    * Skipping (empty) ddb46085053 New check-semgrep-oss.jsonnet workflow (#1506)
    * f089de8d20f ci: pro-benchmark-testing: Rename semgrep -> OSS (#1505)
    * Skipping (empty) 02c488bbc44 Move sync_repos.py to scripts/ and reuiqre it run on develop (#1507)
    * Skipping (empty) 882b17f8ee6 Add targets to build also pysemgrep and run pysemgrep test (#1508)
    * 0fb525f415f Do not ignore OSS/ anymore in precommit (#1509)
    * 014272b1ee6 feat: use `product_ignored_files` in `semgrep ci` (#1501)
    * 55840a1d105 Relax OSS/.pre-commit-config.yaml (#1511)
    * 740a985a8b0 Add more precommit from OSS to pro, part2 (#1512)
    * b273346673b Fix OSS/.../check-semgrep-pro.jsonnet, use OSS folder (#1514)
    * df1b33caeb1 fix: Parsing some git urls that don't end in ".git" (#1504) (#1503)
    * 7476ab56689 precommit part3: adding flake8 from OSS (#1515)
    * Skipping (empty) ad075bd2898 sync: Add sync/sync-with-OSS.sh script to setup the sync
    Success!

### CODE REVIEW

File: perf/run-benchmarks [hunk 1]
```diff
@@ -52,8 +52,8 @@ handler.setFormatter(
 )
 logger.addHandler(handler)
 
-PERF_PATH = "semgrep/perf"
-PERF_RULE_PATH = "semgrep.perf"
+PERF_PATH = "OSS/perf"
```
Comment (aryx) [inline perf/run-benchmarks:55]: @IagoAbal this seems wrong.

File: perf/temp_findings.json [hunk 2]
```diff
@@ -1,4003 +0,0 @@
-[
```
Comment (aryx) [inline perf/temp_findings.json:1]: why was this deleted? @IagoAbal ? @emjin ?

File: perf/snapshots/ci_small_repos_baseline.json [hunk 3]
```diff
@@ -3809,16 +3809,16 @@
...
+          "path": "/Users/emma/workspace/semgrep-proprietary/OSS/perf/bench/kotlinPrefilterTest/input/leakcanary/shark/shark/src/main/java/shark/ReferenceReader.kt",
           "spans": [
             {
               "end": {
                 "col": 46,
                 "line": 4,
                 "offset": 45
               },
-              "file": "/Users/emma/workspace/semgrep-proprietary/semgrep/perf/bench/kotlinPrefilterTest/input/leakcanary/shark/shark/src/main/java/shark/ReferenceReader.kt",
+              "file": "/Users/emma/workspace/semgrep-proprietary/OSS/perf/bench/kotlinPrefilterTest/input/leakcanary/shark/shark/src/main/java/shark/ReferenceReader.kt",
```
Comment (aryx) [inline perf/snapshots/ci_small_repos_baseline.json:3821]: @emjin why do we have /Users/emma/... in snapsot files?

File: perf/temp_findings.json (hunk 2 above)
Comment (IagoAbal) [reply perf/temp_findings.json:1]: I deleted it because given that all paths referred to Emma's HOME path then I assume this was committed by mistake. I warned and asked Emma to review the PR.
